cpp/tests/test_abm_model.cpp

```diff
@@ -380,6 +380,12 @@ TEST(TestModel, evolveMobility)
         auto work_id     = model.add_location(mio::abm::LocationType::Work);
         auto icu_id      = model.add_location(mio::abm::LocationType::ICU);
         auto hospital_id = model.add_location(mio::abm::LocationType::Hospital);
+
+        // Make sure all persons will be InfectedSevere -> InfectedCritical -> Dead.
+        ScopedMockDistribution<testing::StrictMock<MockDistribution<mio::UniformDistribution<double>>>>
+            mock_uniform_dist;
+        EXPECT_CALL(mock_uniform_dist.get_mock(), invoke).WillRepeatedly(testing::Return(1.0));
```

Comment:
> Perhaps we can change the mocking to only affect calls in the infection draw_backwards?

That is quite difficult to do, it would probably be easier (and a cleaner solution) to write the test such that only draw_backwards uses RNG in the first place.

> This seems to be part of 1067 .

I am fine with adding the names of the other tests (with the necessary context) to that issue. We do not need to fix everything here, but at least we have to write down what needs fixing.